migrations/1683556798_addImageCard.ts

```diff
@@ -0,0 +1,141 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
```

Comment:
Could we get rid of this migration? It's replaced by a more recently generated migration in Selwyn's PR